### PR TITLE
fix(70123): fix pagination sales recap --run-build

### PR DESCRIPTION
**Pull request by imam-auliya**
> [#70123] - FE POS - Done
> [#70077] - FE POS - Done
> [#69981] - FE POS - Done
> [#69317] - FE POS - Done
> [#69609] - FE POS - Done

Acceptance Criteria
- [x] Pagination on sales recap
- [x] handle change row
- [x] wording bredcrumb on order menu
- [x] add sales mode id on move item
- [x] add pagination on table select

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=59) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=59)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=59)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=59)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=59)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=59&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications: **Duplication**

File: src/app/modules/order/Components/ListVisitor.components.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ const ListVisitorComponent: React.FC<{
 }> = ({visitors, listVisitor=[], onSelecetVisitor, type}) => {
   const [listVisitorType, setListVisitorType] = useState<IVisitor[]>([])
   useEffect(() => {
+    console.log('LIST VISITOR', listVisitor)
```
Comment (aldiz1711) [inline src/app/modules/order/Components/ListVisitor.components.tsx:13]: bisa di remove ya

**Review by aldiz1711 — APPROVED** [review]
OK

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=59) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=59)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=59)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=59)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=59)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=59&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications: **Duplication**

**Pull request merged**